backend/pkg/sshtunnel/tunnel.go

```diff
@@ -54,6 +56,7 @@ func New(
 			User:            tunnel.User,
 			Auth:            authmethods,
 			HostKeyCallback: getHostKeyCallback(serverPublicKey),
+			Timeout:         30 * time.Second,
```

Comment:
Intentional. Without this it will effectively wait forever, which is not good.
NEOS-1470 to expose this as a configuration option for users